### PR TITLE
Other (table): Reset column widths when table width is reset

### LINKED ISSUE #12916: Removing "columnWidths" property when calling "tableWidth"
Since currently `tableWidth` and `columnWidths` commands are somehow connected ( "columnWidths" property changes when executing `tableWidth` command ), we should also revert "columnWidths" property when executing `tableWidth` without "columnWidths" set.

That means you can call `editor.execute('tableWidth', { tableWidth: '20%', columnWidths: '50%,50%' })` to change both tableWidth and columnWidths, but calling only `editor.execute('tableWidth')` will revert tableWidth, but not columnWidths.

Problem occurred while creating track changes integration for table resize.

### PR DESCRIPTION
**Pull request by kbinieda**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Other (table): Reset column widths when table width is reset. Closes #12916.

### CODE REVIEW

**Review by scofalik — APPROVED** [review]
LGTM.

File: packages/ckeditor5-table/tests/tablecolumnresize/tablewidthresizecommand.js [hunk 1]
```diff
@@ -71,11 +71,11 @@ describe( 'TableWidthResizeCommand', () => {
 		);
 	} );
 
-	it( 'should remove the attribute if new value was not passed', () => {
+	it( 'should remove the attributes if new value was not passed', () => {
```
Comment (arkflpc) [inline packages/ckeditor5-table/tests/tablecolumnresize/tablewidthresizecommand.js:74]: If `tableWidth` and `columnWidths` are not supposed to be connected anymore, I'd add tests for all combinations (missing `tableWidth`, missing `columnWidths`, both missing).

File: packages/ckeditor5-table/src/tablecolumnresize/tablewidthresizecommand.js [hunk 2]
```diff
@@ -51,10 +51,15 @@ export default class TableWidthResizeCommand extends TablePropertyCommand {
 		model.change( writer => {
 			if ( tableWidth ) {
 				writer.setAttribute( this.attributeName, tableWidth, table );
-				writer.setAttribute( 'columnWidths', columnWidths, table );
 			} else {
 				writer.removeAttribute( this.attributeName, table );
 			}
+
+			if ( columnWidths ) {
```
Comment (mlewand) [inline packages/ckeditor5-table/src/tablecolumnresize/tablewidthresizecommand.js:58]: Let's also document the fact that the model attribtue will be removed if respectable option param is not provided.

```
* @param {String} [options.tableWidth] The new table width. Model attribute gets removed if skipped.
```

And same for `options.columnWidths`.

**Review by mlewand — APPROVED** [review]
Just minor things:

*   +1 to [https://github.com/ckeditor/ckeditor5/pull/12917/files#r1034381000](https://github.com/ckeditor/ckeditor5/pull/12917/files#r1034381000)
*   and one minor thing about API docs.

Other than that all good.

**Review by arkflpc — APPROVED** [review]

File: packages/ckeditor5-table/src/tablecolumnresize/tablewidthresizecommand.js [hunk 3]
```diff
@@ -39,8 +39,8 @@ export default class TableWidthResizeCommand extends TablePropertyCommand {
 	 * Changes the `tableWidth` and `columnWidths` attribute values for the given or currently selected table.
 	 *
 	 * @param {Object} options
-	 * @param {String} [options.tableWidth] The new table width.
-	 * @param {String} [options.columnWidths] The new table column widths.
+	 * @param {String} [options.tableWidth] The new table width. Model attribute gets removed if skipped.
+	 * @param {String} [options.columnWidths] The new table column widths. Model attribute gets removed if skipped.
```
Comment (scofalik) [inline packages/ckeditor5-table/src/tablecolumnresize/tablewidthresizecommand.js:43]:
```suggestion
	 * @param {String} [options.tableWidth] The new table width. If skipped, the model attribute will be removed.
	 * @param {String} [options.columnWidths] The new table column widths. If skipped, the model attribute will be removed.
```

**Pull request merged**